packages/sil-voting-app/src/routes/dashboard/components/Proposal/RenderedProposal.tsx

```diff
@@ -0,0 +1,24 @@
+import Hairline from 'medulas-react-components/lib/components/Hairline';
+import React from 'react';
+
+import Proposal, { ProposalProps } from '.';
+
+interface Props {
+  readonly proposal: ProposalProps;
+  readonly index: number;
+}
+
+const RenderedProposal = (props: Props): JSX.Element => {
```

Comment:
We are usually using object destructuring for props properties.

Reply:
Thanks for the advice, I changed every component to follow that approach.